policyengine_us/variables/gov/states/la/tax/income/exempt_income/la_agi_exempt_income.py

```diff
@@ -6,6 +6,7 @@ class la_agi_exempt_income(Variable):
     entity = TaxUnit
     label = "Louisiana income that is exempt from the adjusted gross income"
     defined_for = StateCode.LA
+    reference = "https://revenue.louisiana.gov/TaxForms/IT540i(2021)%20Instructions.pdfpage=9"
```

Comment:
can you add a legal code reference?

Reply:
The legal code section has been repealed and I cannot find the text pre 2022